paymentsheet/res/values/donttranslate.xml

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Was this just a merge error?  I think we still want this file to hold the strings that will not be in our localization solution.

Reply:
I renamed to `donottranslate.xml`, that's the file name that means something to Android and it won't look for translated strings. Then we can remove `tools:ignore="MissingTranslation"`.